code/index.html

```diff
@@ -1,22 +1,189 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+
+<head>
+  <meta charset="utf-8" />
+  <title>🌵 CACTUS NEWS</title>
+  <link rel="stylesheet" href="./style.css" />
+  <link href="https://fonts.googleapis.com/css?family=Gelasio&display=swap" rel="stylesheet">
+  <meta name="viewport" content="width=device-width">
+</head>
+
+<body>
+
+  <!-- Main Content -->
+  <main>
+    <!-- Header -->
+    <header class="header">
+      <h1>
+        🌵 CACTUS NEWS
+      </h1>
+
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+
+      <nav>
+        <a href="#">NEWS</a>
+        <a href="#">CACTUS OF THE WEEK</a>
+        <a href="#">WEATHER</a>
+        <a href="#">CONTACT</a>
+      </nav>
+
     </header>
 
+    <!-- big news -->
     <section class="big-news">
-      <!-- The big news content -->
+      Big news
+      <div>
+        <img src="pictures/cactus-farm.jpg" alt="cactus-farm">
```

Comment:
The `alt` text are written as "normal" text :)
```suggestion
        <img src="pictures/cactus-farm.jpg" alt="cactus farm">
```